src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -160,6 +164,10 @@ def creatorProcess(work, jobCacheDir):
         swVersion    = work.get('swVersion', None)
         agentNumber  = work.get('agentNumber', 0)
         numberOfCores = work.get('numberOfCores', 1)
+        inputDataset = work.get('inputDataset', None)
+        inputDatasetLocations = work.get('inputDatasetLocations', None)
+        logging.warning("ALEX JobCreator 5: inputDataset %s" % inputDataset)
```

Comment:
Make sure these are eventually removed, even if it's some time after this goes into production. If you want to put it into production like this, please open a ticket now to remind yourself.